cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
The `tests.py` file is empty. Consider adding test cases to verify the functionality of your models, views, and serializers. This will help ensure that your application behaves as expected and can handle edge cases.